### PR TITLE
NF: remove toLong and toInt from equal assertions

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Mostly fixes bug: #11965, except for examples such as https://github.com/ankidroid/Anki-Android/blob/main/AnkiDroid/src/test/java/com/ichi2/anki/DeckPickerTest.kt#L160 where, if you remove the toLong you get the error message:
> java.lang.AssertionError: expected: java.lang.Integer<20800301> but was: java.lang.Long<20800301>

### CODE REVIEW

File: AnkiDroid/src/main/res/values/attrs.xml [hunk 1]
```diff
@@ -144,6 +144,9 @@
     <!-- Introduction -->
     <attr name="introductionLogoTransparent" format="color"/>
 
+    <!-- Deck Notification UI-->
+    <attr name="notificationPreferenceProgressbarColor" format="color" />
+
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/res/values/attrs.xml:149]: unrelated

File: AnkiDroid/src/test/java/com/ichi2/anki/preferences/SettingsSearchBarTest.kt [hunk 2]
```diff
@@ -39,27 +38,27 @@ class SettingsSearchBarTest : RobolectricTest() {
         // Use reflection to access some private fields
         val filesToIndexField = getJavaFieldAsAccessible(SearchConfiguration::class.java, "filesToIndex")
         val searchItemResIdField = getJavaFieldAsAccessible(SearchConfiguration.SearchIndexItem::class.java, "resId")
-        val preferencesToIndexField = getJavaFieldAsAccessible(SearchConfiguration::class.java, "preferencesToIndex")
-        val prefItemResIdField = getJavaFieldAsAccessible(PreferenceItem::class.java, "resId")
+//        val preferencesToIndexField = getJavaFieldAsAccessible(SearchConfiguration::class.java, "preferencesToIndex")
```
Comment (BrayanDSO) [inline AnkiDroid/src/test/java/com/ichi2/anki/preferences/SettingsSearchBarTest.kt:41]: all changes on this file are unrelated

**Review by BrayanDSO — CHANGES_REQUESTED** [review]
I believe that you mixed your branches by mistake

**Comment by Arthur-Milchior** [comment]
strange. But you're totally right. I got something wrong here

**Pull request closed without merging**